packages/db/src/schema/posts.ts

```diff
@@ -0,0 +1,31 @@
+import {
+	pgTable,
+	text,
+	timestamp,
+	jsonb,
+	primaryKey,
+	boolean,
+} from "drizzle-orm/pg-core";
+import { profiles } from "./profiles.ts";
+
+export const posts = pgTable(
+	"posts",
+	{
+		slug: text("slug").notNull(),
+		locale: text("locale").notNull(),
+		title: text("title").notNull(),
+		description: text("description"),
+		socialImage: text("social_image"),
+		bannerImage: text("banner_image"),
+		version: text("version"),
+		originalLink: text("original_link"),
+		noindex: boolean("noindex"),
+		editedAt: timestamp("edited_at"),
+		publishedAt: timestamp("published_at").$default(() => new Date()),
+		meta: jsonb("meta"),
+		authorSlug: text("author_slug")
+			.notNull()
+			.references(() => profiles.slug),
```

Comment:
This should be a one-to-many relation (there can be multiple authors per post), so it will need a `post_authors` table.